selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindReferencesConsoleTab.java

```diff
@@ -93,6 +107,12 @@ public void doubleClickOnReference(String visibleText) {
     seleniumWebDriverHelper.doubleClick();
   }
 
+  public void doubleClickOnReferenceWithTextEqualsTo(String visibleText) {
```

Comment:
IMHO next method signature will be more relevant to what the method is actually does:
```
doubleClickOnReferenceEqualsTo(String referenceText)
```